### PR TITLE
Products

### PR DESCRIPTION
**Pull request by Developers-codz**
## Files To Check 

`src\functions\get-filtered-data.js`
`src\functions\get-sorted-data.js`
`src\pages\product\Product.jsx`
`src\reducer\product-reducer.js`
`src\data\product-list.js`
`src\App.js`

Added product page and sort filter clear all functionality basically also added routes and link for switching between the pages
Thank you for reviewing my PR :)
Capture1

Capture2

### CODE REVIEW

File: src/functions/get-filtered-data.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const getFilteredData = (data, categoryBy, clear) => {
+  if (categoryBy && categoryBy === "FILTER_BY_PAINTING") {
```
Comment (arjun-computer-geek) [inline src/functions/get-filtered-data.js:2]: It would be better if constants are in the constant folder

**Review by arjun-computer-geek — APPROVED** [review]
Else looks good

**Review by vashnavichauhan1825 — APPROVED** [review]
LGTM . approved for merge

File: src/functions/get-filtered-data.js (hunk 1 above)
Comment (Developers-codz) [reply src/functions/get-filtered-data.js:2]: Ok will look into that

**Pull request merged**